var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -85,17 +93,46 @@ def languages(self):
 
     def configure_args(self):
         spec = self.spec
-        cc = self.compiler.cc
-        cxx = self.compiler.cxx
+
+        if '+intel_backend' in spec:
+            cc = which('mpicc')
+            cxx = which('mpic++')
+        else:
+            cc = self.compiler.cc
+            cxx = self.compiler.cxx
```

Comment:
You definitely don't want to do this. `self.compiler.cc` is the actual compiler, so you would be bypassing Spack's compiler wrappers. Use `spack_cc` instead. Unless I'm missing something. I'm not that familiar with cross-compilation.